Sources/swift-watch/Logger.swift

```diff
@@ -0,0 +1,97 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+
+import Foundation
+
+class Logger {
+    let configuration: Configuration
+
+    fileprivate var colored: Bool {
+        return !self.configuration.monochrome
+    }
+
+    fileprivate var notQuiet: Bool {
```

Comment:
Why `notQuiet` instead of `quiet`?

Reply:
The command-line option is opt-out. In my code however I want to check for opt-in.

Have a better idea how to get this cleanly? Was thinking about `var isLoud: Bool`, but that wouldn't really convey the intent properly.

Reply:
Maybe `isVerbose`